text/3336-maybe-dangling.md

```diff
@@ -3,7 +3,7 @@
 - Feature Name: `maybe_dangling`
 - Start Date: 2022-09-30
 - RFC PR: [rust-lang/rfcs#3336](https://github.com/rust-lang/rfcs/pull/3336)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Tracking Issue: [rust-lang/rust#118166](https://github.com/rust-lang/rust/issues/118166)
```

Comment:
how about also fixing the template which currently still reads "Rust Issue:" here